src/EditorFeatures/CSharpTest/CodeActions/ConvertLinq/ConvertLinqQueryToForEachTests.cs

```diff
@@ -0,0 +1,2619 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeActions.ConvertLinq
+{
+    public class ConvertLinqQueryToForEachTests : AbstractCSharpCodeActionTest
```

Comment:
>ConvertLinqQueryToForEachTests [](start = 17, length = 30)

It doesn't look like there are any tests for queries targeting IQueryable

Reply:
Can we consider them in a separate PR?

---
In reply to: [177198499](https://github.com/dotnet/roslyn/pull/25362#discussion_r177198499) [](ancestors = 177198499)

Reply:
> Can we consider them in a separate PR?

I think we should have at least two positive tests for IQueryable in this PR. I described them in a comment within AsQueryable unit-test.

---
In reply to: [178426483](https://github.com/dotnet/roslyn/pull/25362#discussion_r178426483) [](ancestors = 178426483,177198499)

Reply:
Adding some more tests. Consider support in [https://github.com/dotnet/roslyn/issues/25639](https://github.com/dotnet/roslyn/issues/25639)

---
In reply to: [178426483](https://github.com/dotnet/roslyn/pull/25362#discussion_r178426483) [](ancestors = 178426483,177198499)